src/app/pages/common/entity/entity-dialog/entity-dialog.component.css

```diff
@@ -4,4 +4,11 @@ button.action-btn {
 
 div.entity-dialog-form-message {
     margin: 0 0 13px;
+}
+
+.entity-dialog-form-content {
+    overflow-y: scroll !important;
```

Comment:
It means scrollbar will be visible always even if the content fits without scrolling, isn't?

Reply:
Nice catch. Fixed.